### ISSUE TITLE
[Bug]: Auth::user() - autocomplete does not work

### ISSUE THREAD

**Issue by Webotvorba** [laravel-idea/plugin#965]
### Bug description

After update to Laravel 11 the autocomplete doesnt work for Auth::user() properties

![Snímka obrazovky 2024-03-23 o 11 21 11](https://github.com/laravel-idea/plugin/assets/25526581/eb61c63e-a09c-4eaa-9723-e645ad9bc9a1)

### Plugin version

8.1.3.233

### Operating system

MacOS

**Comment by Webotvorba** [comment]
Also for $request variable

```
public function index(Request $request): View
    {
        $vendor = $request->user()->name;
        return view('merchant.report_technology', [
            'title' => __('titles.merchant.disposition'),
            'view' => '',
            'vendor' => Auth::user()->vendor,
            'technology' => Auth::user()->vendor->technology->where('problem', 1),
        ]);

    } 
```

**Comment by adelf** [comment]
Fixed. Thanks. The update will be available soon.

**Comment by Webotvorba** [comment]
Any time estimate for the fixed release? Thanks

**Comment by adelf** [comment]
next week.

**Comment by adelf** [comment]
Fixed in 8.1.4
